libr/anal/type.c

```diff
@@ -122,18 +122,107 @@ R_API void r_anal_save_parsed_type(RAnal *anal, const char *parsed) {
 static int typecmp(const void *a, const void *b) {
 	char *type1 = (char *) a;
```

Comment:
this is const char * remove the cast

Reply:
Oh, that's not the code I wrote, but I guess I can still remove it

Reply:
So I can call strcmp directly with a and b?